### PR TITLE
docs: ✏️ Gitlab info for stronghold. Changed oscar account info

### PR DESCRIPTION
**Pull request by prasadbandarkar**
Added a line describing a new feature on stronghold : Gitlab . Fixed the
rates documentation for GPU priority+ Oscar account.

BREAKING CHANGE: 🧨 None

✅ Closes: None

### Pull Request
---

#### PR Summary
Briefly describe the changes.

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [ ] :frog: Data (data folder - people/opportunities)
- [x] :dog: Content (content folder) : content/english/services/infrastructure/stronghold.md
- [x] :blowfish: Other. Specify: themes/gb-theme/layouts/shortcodes/

#### Checklist:
- [ ] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [ ] Contributing guidelines were followed.
- [ ] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

**Comment by paulstey** [comment]
Thanks, @prasadbandarkar!! This looks good to me. 

Just out of curiosity, @mcmcgrath13, why didn't this build an ephemeral version of the site in Heroku? Is that Action broken?

**Comment by mcmcgrath13** [comment]
looking at heroku I don't see anything obvious.  I wonder if it's because the PR is from a fork instead of a branch

**Comment by mcmcgrath13** [comment]
@prasadbandarkar in the future, you can/should branch off this repo instead of making a fork (and we'll see if that was the issue 🤔 )

File: content/english/services/infrastructure/stronghold.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ Each Principal Investigator (PI) is given a dedicated environment for their proj
 - **Environments** - Linux or Windows compute environments available
 - **Software** - Workstations come preinstalled with our Windows Standard Software or Linux Standard Software packages
 - **Automated Git Sync** - Changes made to remote git repositories are automatically pulled to local repositories in Stronghold after initial configuration
+- **Gitlab** - Host and manage git repositories within the secure environment
```
Comment (mcmcgrath13) [inline content/english/services/infrastructure/stronghold.md]: How is this different from the `**Automated Git Sync**` above?  Is there gitlab server running inside stronghold?

Comment (paulstey) [reply content/english/services/infrastructure/stronghold.md]: Yes, that is correct. There is an internal GitLab in Stronghold as of very recently.

Comment (mcmcgrath13) [reply content/english/services/infrastructure/stronghold.md]: Cool! I think these two bullet points need to be more differentiated to make that/those use cases clearer

**Review by mcmcgrath13 — APPROVED** [review]

**Review by paulstey — APPROVED** [review]

**Comment by prasadbandarkar** [comment]
Can this pull request be approved today. Let me know if something else is required from my end.
Thank you.

**Pull request merged**